backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java

```diff
@@ -276,6 +276,7 @@ private Project insertProject(Project project, String workspaceName, UserInfo us
   public Project updateProjectName(UserInfo userInfo, String projectId, String projectName)
       throws InvalidProtocolBufferException {
     try (Session session = ModelDBHibernateUtil.getSessionFactory().openSession()) {
+      // TODO: Remove this after UAC support update entity name using SetResource
```

Comment:
@conradoverta added TODO for how to update entity name at UAC??

Reply:
UAC already supports name update, but the question here is more how to ensure consistency between the two systems

Reply:
I am trying to figure out how to update resource name at UAC. thread https://vertaai.slack.com/archives/C01JVHUHKR6/p1610115155062800

Reply:
Not a blocker to merge this